### ISSUE TITLE
Let's add this content to developer.overheid.nl

### ISSUE THREAD

**Issue by ehotting** [J535D165/PublicSectorNL#12]
https://developer.overheid.nl is meant to be a portal for the target audience 'developers' working inside or with Dutch government. Of course it's open source itself: https://gitlab.com/commonground/developer.overheid.nl

This list with all public organisations and their open source repositories would fit there nicely. Currently some work is done to implement a 'code' feature, see https://don-feature-236-code-sz7ran.nlx.reviews for work in progress.

I propose to discuss how this content could be represented there in a useful way, reaching a larger audience perhaps, and being enhanced with visual design and features (like tracing activity in repos, integrating with community Discourse, and so on).

**Comment by J535D165** [comment]
Thanks, we are happy to integrate this list with https://developer.overheid.nl. I like the idea of an enhanced design, a search option, and adding more metadata. With the GitHub and GitLab api's we can make al repos findable, which would be very nice to have. 

What is a good way to start working on this? I'm not a frontend dev, but I can provide some help on creating a structured, machine-readable version of this repo. We can create a standalone repo at the commonground GitLab for this if you like. This can serve the database or frontend. Any thoughts on this?

**Comment by ehotting** [comment]
Creating the feature on developer.overheid.nl, including some design work etc. is not the hardest part. That is a one time investment. Most important is to have a community of people that keep adding / updating the content. I think we need at least:

1. machine readable repo which anyone can propose changes to using PR's
   - would be nice to create this together. Happy to give you rights on developer.overheid.nl repo. Something similar happens here: https://gitlab.com/commonground/developer.overheid.nl/-/tree/master/data/apis but perhaps that should be restructured as well and moved to a separate repo for content
1. pool of people who are willing and able to judge the PR's and redact / accept / deny them
   - people like you from the broader community that have maintainer rights on repo
   - some kind of policy will develop naturally and can be codified over time, which repo's pass the bar to be included
1. section where this content is presented in a nice way
   - I can provide UX/Design capability to make sure it fits in current concept
   - We should brainstorm with OSF how this releates to 'code' feature, can be a video call
   - Implementation of frontend can be done by my team (or anyone else that knows React - we use our [design system](https://gitlab.com/commonground/core/design-system) so should be pretty straight forward

That's where we could start.
Once that is in place, we can extend. Some features that come to mind (but I'd like to confront this with user need):

1. Multi-lingual  (would be nice to include this in machine readable format from the start)
1. Present them structured based on certain view points, including search (similar to https://developer.overheid.nl/apis but with more advanced structure, challenge for designer)
   - Default structure can for now be hard coded based on how organisations related to each other, but potentially relations between organisations could be shown (parent / child / cooperation)
   - Choose from limited list of themes (or work with tags) - implications for machine readable format
1. Validate organisations and show this validation
1. Show activity of an organisation (sky is the limit)

Some other ideas:
1. If the machine readable format is a json per organisation / collection of repositories, this could be hosted anyware if we combine it with a fixed filename file that can be found anywhere on github/gitlab using their API's, comparable to https://github.com/italia/publiccode.yml  <- perhaps we can even use exactly this
